beagleboard/librobotcontrol
# V1.0.5 merge

* use latest aliases for PRU indexes
* add support for PocketBeagle
* integrate PX4 build handling
* clean-up some messages from rc_balance
* fix rc_startup_routine start delay
* fix rc_filter_duplicate initialization check
* fix stdin control mode for rc_balance
* add PRU to startup routine checks and required kernel modules
